docs/tutorials/06-batch-infer.ipynb

```diff
@@ -1240,7 +1293,7 @@
    "id": "4b1fe08a",
    "metadata": {},
    "source": [
-    "Enabling model monitoring application with default `deploy_histogram_data_drift_app` provide additional plots like; `drift_table_plot` that compares the drift between the training data and prediction data per feature and `features_drift_results` to show each feature drift score."
+    "The custom histogram data drift model monitoring application we deployed provides additional plots like; `drift_table_plot` that compares the drift between the training data and prediction data per feature and `features_drift_results` to show each feature drift score."
```

Comment:
"The custom histogram data drift model monitoring application you deployed provides additional plots like: `drift_table_plot` that compares the drift between the training data and prediction data per feature; and `features_drift_results` to show each feature drift score."